src/Microsoft.ML.Data/Transforms/TermTransform.cs

```diff
@@ -29,14 +29,14 @@
 
 namespace Microsoft.ML.Runtime.Data
 {
-    /// <summary>
-    /// TermTransform builds up term vocabularies (dictionaries).
-    /// Notes:
-    /// * Each column builds/uses exactly one "vocabulary" (dictionary).
-    /// * Output columns are KeyType-valued.
-    /// * The Key value is the one-based index of the item in the dictionary.
-    /// * Not found is assigned the value zero.
-    /// </summary>
+
+    // TermTransform builds up term vocabularies (dictionaries).
+    // Notes:
+    // * Each column builds/uses exactly one "vocabulary" (dictionary).
+    // * Output columns are KeyType-valued.
+    // * The Key value is the one-based index of the item in the dictionary.
+    // * Not found is assigned the value zero.
```

Comment:
We'll have to be careful here. Conceptually, the [key-values](https://github.com/dotnet/machinelearning/blob/master/docs/code/KeyValues.md) are *logically* starting at 0, but physically valid values start at 1. I feel like this might not be the best place to talk about key-values unless you're *really* going to go into them, since otherwise it may be confusing.

Reply:
Leaving it as is, since it is just a code comment for us.  

---
In reply to: [202454960](https://github.com/dotnet/machinelearning/pull/529#discussion_r202454960) [](ancestors = 202454960)